src/main/java/de/retest/recheck/RecheckImpl.java

```diff
@@ -58,7 +62,7 @@
 
 	private final Map<String, DefaultValueFinder> usedFinders = new HashMap<>();
 	private final TestReplayResultPrinter printer;
-	private final GlobalIgnoreApplier ignoreApplier;
+	private final Filter ignoreApplier;
```

Comment:
I think changing the field name would also be a good idea:

```suggestion
	private final Filter filter;
```